backend/packages/identity-scan/src/scan/utils/getIdentityStorage.js

```diff
@@ -0,0 +1,48 @@
+const {
+  chain: { getApi },
+} = require("@osn/scan-common");
+const { toDecimal } = require("./unitConversion");
+
+async function getIdentityStorage(accountId) {
+  const api = await getApi();
+  let identity = {};
+  identity.accountId = accountId;
+
+  const identityInfo = await api.query.identity.identityOf(accountId);
+
+  if (!identityInfo.isSome) {
+    return identity;
+  }
+
+  const { info, judgements, deposit } = identityInfo.unwrap();
+  identity.info = {
+    display: info.display.asRaw.toUtf8(),
+    legal: info.legal.asRaw.toHuman(),
+    web: info.web.asRaw.toHuman(),
+    riot: info.riot.asRaw.toHuman(),
+    email: info.email.asRaw.toHuman(),
+    image: info.image.asRaw.toHuman(),
+    twitter: info.twitter.asRaw.toHuman(),
+    pgpFingerprint: info.pgpFingerprint.isSome
+      ? info.pgpFingerprint.unwrap().toHex()
+      : null,
+  };
+  identity.deposit = await toDecimal(deposit.toNumber());
+  if (judgements.length > 0) {
+    let judgementsList = [];
+    judgements.forEach(([registrarIndex, judgement]) => {
+      let judgementInfo = {
+        registrarIndex: registrarIndex.toNumber(),
+        judgement: judgement.toString(),
+      };
+      judgementsList.push(judgementInfo);
+    });
+    identity.judgements = judgementsList;
+  }
```

Comment:
1. I suggest extract judgements extraction to a separate method to keep the main function short.
2. Why not use `map` instead of `foeEach`. Upper code block looks verbose.